mlrun/db/httpdb.py

```diff
@@ -1412,14 +1414,25 @@ def submit_pipeline(
         :param namespace: Kubernetes namespace to execute the pipeline in.
         :param artifact_path: A path to artifacts used by this pipeline.
         :param ops: Transformers to apply on all ops in the pipeline.
-        :param ttl: Set the TTL for the pipeline after its completion.
-        """
+        :param ttl: pipeline cleanup ttl in secs (time to wait after workflow completion, at which point the workflow
+                    and all its resources are deleted) (deprecated, use cleanup_ttl instead)
+        :param cleanup_ttl: pipeline cleanup ttl in secs (time to wait after workflow completion, at which point the
+                            workflow and all its resources are deleted)
+        """
+
+        if ttl:
+            warnings.warn(
+                "ttl is deprecated, please use cleanup_ttl instead",
+                "This will be removed in 1.5.0",
+                # TODO: Remove this in 1.5.0
+                PendingDeprecationWarning,
```

Comment:
`PendingDeprecationWarning` is not displayed to users
Also we decided to not be polite 😄 
```suggestion
                "'ttl' is deprecated, use 'cleanup_ttl' instead",
                "This will be removed in 1.5.0",
                # TODO: Remove this in 1.5.0
                FutureWarning,
```